content/docs/tasks/policy-enforcement/rate-limiting.md

```diff
@@ -20,10 +20,40 @@ service.
   requests from test user "jason" to version v2 and requests from any other
   user to v3.
 
-    ```command
-    $ istioctl create -f @samples/bookinfo/kube/route-rule-reviews-test-v2.yaml@
-    $ istioctl create -f @samples/bookinfo/kube/route-rule-reviews-v3.yaml@
-    ```
+  ```command
+  $ istioctl create -f @samples/bookinfo/routing/route-rule-all-v1.yaml@
+  ```
+
+  Save the following YAML snippet as `route-rule-reviews-jason-v2-v3.yaml.yaml`:
```

Comment:
As in the other PR, that's a mighty suffix to the file name...

Reply:
Done